Another revision of uv-managed branch (#2074)
plone/documentation: docs/admin-guide/override-core.md

## Override the version of a core Plone package

> The Python packages on which Plone depends are pinned to specific versions at the time a Plone release is created.

Reviewer: I intentionally phrased it to use the word "dependencies" which is more likely to be recognized in this context. But I don't feel strongly.

Maybe could be simpler:
```suggestion
Plone's Python package dependencies are pinned to specific versions at the time a Plone release is created.
```

Author: I like. Yeah, the original was an awkward mouthful.
